packages/ui/src/utils/ThemeUtil.ts

```diff
@@ -40,7 +40,7 @@ export function setThemeVariables(themeVariables: ThemeVariables) {
   }
 }
 
-function createRootStyles(themeVariables?: ThemeVariables) {
+export function createRootStyles(themeVariables?: ThemeVariables) {
```

Comment:
Do we need to export it with this solution?

Reply:
Nope, let me revert